public/index.html

```diff
@@ -8,50 +8,25 @@
     <meta name="description" content="Useful app to check how you're getting on in hacktoberfest">
     <meta property="og:title" content="Hacktoberfest Checker">
     <meta property="og:description" content="Useful app to check how you're getting on in hacktoberfest">
-    <link rel="canonical" href="{{ hostname }}">
-    <meta property="og:url" content="{{ hostname }}">
+    <link rel="canonical" href="%REACT_APP_HOSTNAME%">
+    <meta property="og:url" content="%REACT_APP_HOSTNAME%">
     <meta property="og:site_name" content="Hacktoberfest Checker">
-    <meta property="og:image" content="{{ hostname }}/img/og.png">
+    <meta property="og:image" content="%PUBLIC_URL%/img/og.png">
```

Comment:
This is something provided by react scripts right? Does it differ from `REACT_APP_HOSTNAME`? Should we just use `REACT_APP_HOSTNAME`?

Reply:
PUBLIC_URL is specifically for stuff in the public folder :smile:

Reply:
I think we should use the hostname always, some people check out this repo and run it in sub folders and things and can see this potentially being troublesome.

Reply:
As far as I'm aware it's actually a relative path but I'm not sure...
Will change it to REACT_APP_HOSTNAME, as well as replace the use of window.location.origin with REACT_APP_HOSTNAME.